key.util/src/main/java/org/key_project/util/java/StringUtil.java

```diff
@@ -167,7 +174,7 @@ public static boolean contains(String string, CharSequence substring) {
      *
      * @author Mattias Ulbrich (under GPL)
      */
-    public static String wrapLines(String string, int length) {
+    public static @NonNull String wrapLines(@NonNull String string, int length) {
```

Comment:
There is no need to make `@NonNull` explicit here and below. In general, `@NonNull` is usually only needed on type variables.

Reply:
Yes, I know. But I somehow thought that in a semi-annotated context having explicit `@NonNull` would have a documentation benefit. To be discussed.